job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/step/StepContext.groovy

```diff
@@ -651,6 +651,15 @@ class StepContext implements Context {
                             configs('class': 'java.util.Collections$EmptyList')
                         }
                     }
+
+                    // Apply overrides
+                    if (jobInPhase.configureClosure) {
+                        jobInPhase.configureClosure.resolveStrategy = Closure.DELEGATE_FIRST
```

Comment:
Because I copied the extendedEmail strategy, lol.  I'll fix right now.  Thank you.